src/main/java/com/uid2/shared/encryption/AesCbc.java

```diff
@@ -54,4 +55,36 @@ public static byte[] decrypt(byte[] encryptedBytes, SecretKey key) {
             throw new RuntimeException("Unable to Decrypt", e);
         }
     }
+
+    // TODO: after KeySets fully migrated, below APIs shall be removed.
+    public static EncryptedPayload encrypt(byte[] b, EncryptionKey key) {
```

Comment:
is there a way to avoid the code duplication?

Reply:
Refactored the multiple encrypt and decrypt functions.

Reply:
Let's have a single encrypt with try/catch, and a single decrypt with try/catch.
The other overloads should be 1 line only

Reply:
Refactored encrypt/decrypt to single one with try/catch. Removed redundant try/catch.